jdaviz/configs/default/plugins/line_lists/line_lists.vue

```diff
@@ -94,6 +94,7 @@
     <j-plugin-section-header>Preset Line Lists</j-plugin-section-header>
     <v-row>
       <v-select
+        :menu-props="{ left: true }"
```

Comment:
we should probably do this app-wide... how does this work in conjunction with #1673?

Reply:
Good point @kecnry! I've implemented it across all v-selects in https://github.com/spacetelescope/jdaviz/pull/1656/commits/bfa270fb99f5761ef13ff493bcbefdf581302c05

The short answer to your question about how they interact together is that they don't. I WISH they did. Unfortunately #1673 restricts menus to the size of the v-select when you attach. We'll have to think through this case before we merge #1673. I'd advocate for us to get this in as it stands and think about those considerations over in that PR